client/html/src/Client/Html/Basket/Standard/Standard.php

```diff
@@ -495,47 +497,30 @@ protected function addProducts( \Aimeos\MW\View\Iface $view )
 
 			for($i = 0; $i < count($entries); ++$i) {
 			    $paths = [];
-                if (isset( $entries[$i]['attrcustid'][$attrId] ) && is_array($entries[$i]['attrcustid'][$attrId])) {
-                    /** @var UploadedFile $file */
-                    foreach($entries[$i]['attrcustid'][$attrId] as $file) {
-                        $filepath = 'basket-upload/' . md5($file->getFilename() . microtime(true)) . '.' . $file->extension();
-                        try {
-                            $stream = fopen($file->getRealPath(), 'r+');
-                            $fs->writes($filepath, $stream);
-                            fclose($stream);
-                        } catch (\Exception $ex) {
-                            Log::error($ex->getMessage());
-                        }
-                        $paths[] = $filepath;
-                    }
-                    $entries[$i]['attrcustid'][$attrId] = $paths;
-                }
-                if( isset( $entries[$i]['prodid'] ) ) {
-                    $list[] = $entries[$i]['prodid'];
-                }
-            }
-			/*foreach( $entries as $values )
-			{
-                if (isset( $values['attrcustid'][$attrId] )) {
-                    $file = $values['attrcustid'][$attrId];
-                    $filepath = 'basket/' . md5($file->getFilename() . microtime(true)) . '.' . $file->extension();
-                    try {
-                        $stream = fopen($file->getRealPath(), 'r+');
-                        $fs->writes($filepath, $stream);
-                        fclose($stream);
-                    } catch (\Exception $ex) {
-                        Log::error($ex->getMessage());
-                    }
-                    $values['attrcustid'][$attrId] = $filepath;
-                }
-				if( isset( $values['prodid'] ) ) {
-					$list[] = $values['prodid'];
+				foreach ($attrIds as $attrId) {
+					if (isset($entries[$i]['attrcustid'][$attrId]) && is_array($entries[$i]['attrcustid'][$attrId])) {
+						/** @var UploadedFile $file */
+						foreach ($entries[$i]['attrcustid'][$attrId] as $file) {
+							$filepath = 'basket-upload/' . md5($file->getFilename() . microtime(true)) . '.' . $file->extension();
+							try {
+								$stream = fopen($file->getRealPath(), 'r+');
+								$fs->writes($filepath, $stream);
+								fclose($stream);
+							} catch (\Exception $ex) {
+								Log::error($ex->getMessage());
```

Comment:
`Log::error()` is also Laravel specific. Use `$this->getContext()->logger()->log()` instead